TACC/Core-Portal
# task/FP-739: Add Make Public button.

## Overview: ##
Add a toolbar button that copies files directly to the Public Data system (This is a feature 3DEM requested).
## Related Jira tickets: ##

* [FP-739](https://jira.tacc.utexas.edu/browse/FP-739)

## Summary of Changes: ##

## Testing Steps: ##
1. Update your settings_secret: 

> _WORKBENCH_SETTINGS = {
    "debug": _DEBUG,
    "viewPath": True,
    "makeLink": True,
    "makePublic": True
}

2. Go to My Data, select a file and hit Make Public. Make sure that the file is copied and notification toasts pop up.

## UI Photos:
![image](https://user-images.githubusercontent.com/12601812/105402478-16a1a080-5bed-11eb-8208-b57150c0c577.png)

## Notes: ##
